public/app/features/explore/Explore.tsx

```diff
@@ -66,6 +66,7 @@ const getStyles = stylesFactory(() => {
       label: logsMain;
       // Is needed for some transition animations to work.
       position: relative;
+      margin-top: 21px;
```

Comment:
this value seems arbitrary, should it not be a multiple of our spacing grid (8px) (using spacing.md)

Reply:
Funny, I asked Andrej about this just before. Good to learn that you do use a spacing concept like that! 🙂

Although I've seen a few 4 px values too - which makes a lot of sense, since the grafana UI is relatively dense in places...

Reply:
Well no idea it's from the `m-t-2` class that was used there before. If that needs changing would probably make sense to change it separately, maybe as kind of "unify spacing" task